frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -13,20 +13,20 @@ export type CheckboxProps = {
 };
```

Comment:
Refactoring this file to allow for a column-centered checkbox layout as needed by the syphilis test card. Splitting this out here so that it's a bit easier to review

Reply:
I think this is alright since it's just a representational change of the checkboxes rather than additional complex functionality (like comparing it to the `MultiSelectDropdown` which is a bit unwieldy)